katka/models.py

```diff
@@ -124,6 +124,7 @@ class Meta:
     stage = models.CharField(max_length=100)
     status = models.CharField(max_length=30, choices=STEP_STATUS_CHOICES, default=STEP_STATUS_NOT_STARTED)
     output = models.TextField(blank=True)
+    sequence_id = models.CharField(max_length=30, blank=True, null=True)
```

Comment:
shall we put some text explaining how this sequence works? I just don't know where is the best place to put it

Reply:
I added some comments to clarify it :)